packages/@n8n/config/src/configs/runners.config.ts

```diff
@@ -53,4 +60,12 @@ export class TaskRunnersConfig {
 	/** Should the output of deduplication be asserted for correctness */
 	@Env('N8N_RUNNERS_ASSERT_DEDUPLICATION_OUTPUT')
 	assertDeduplicationOutput: boolean = false;
+
+	/** How long (in minutes) until shutting down an idle runner. */
+	@Env('N8N_RUNNERS_IDLE_TIMEOUT')
+	idleTimeout: number = 5;
```

Comment:
Since JS doesn't have the concept `unit` I personally I usually like to add the unit to the name of the variable so it's explicit what the unit is. E.g. This could be `idleTimeoutInMins` or `idleTimeoutInMinutes`. Same for the check frequency